setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-setup(name='bq2dbt', version='0.1.0',
+setup(name='bq2dbt', version='0.1.1',
```

Comment:
We should try to make this dynamic.

Next version will be 0.3.0 actually.

Reply:
I don't no know to do that :) Get current tag value ?